src/api/controllers/PostController.ts

```diff
@@ -1,7 +1,14 @@
 import { Body, Delete, Get, JsonController, Params, Post } from 'routing-controllers';
 
 import { PostService } from '../../services/PostService';
-import { CreatePostRequest, ErrorResponse, getErrorMessage, GetPostResponse, GetPostsResponse, getSavedPostsRequest } from '../../types';
+import {
+  CreatePostRequest,
+  ErrorResponse,
+  getErrorMessage,
```

Comment:
I just noticed these, should be CapitalCase not camelCase

Reply:
the camelCase is for functions, CapitalCase for classes/interfaces